components/typography/typography.css

```diff
@@ -0,0 +1,75 @@
+.typography{
+    display: flex;
+    flex-direction: column;
+}
+
+.example-display-text{
+    border: 1px solid var(--secondary-color);
+    border-radius: 4px;
+    background-color: var(--off-white);
+    min-height: 100px;
+    width: 100%;
+    padding: 1rem;
+    margin-bottom: 1.5rem;
+}
+
+h1, h2, h3, h4, h5, h6{
+    padding: 0.5rem;
+}
```

Comment:
Wanted to know the reason behind selecting the elements is it because where you use these heading tags by default a padding of 0.5rem must be given?
My suggestion would be Instead of selecting the elements you can directly create a class named as pad-0-5 which has this style declaration within it and then use it in these headings.

Reply:
Yes, will make that change.